<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DCM</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>271</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Patient Information</string>
    </property>
    <widget class="QLineEdit" name="Name">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Age">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ID">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="StudyData">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ContentData">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>200</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>姓 名 ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>年 龄 ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>病 人 ID：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>检查时间：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>拍片时间：</string>
     </property>
    </widget>
    <widget class="QLabel" name="NameException">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>42</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="AgeException">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>82</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="IDException">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>122</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="StudyDataException">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>162</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ContentDataException">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>202</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Save">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Reset">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>DCM Image</string>
    </property>
    <widget class="QLabel" name="DCMPaint">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>331</width>
       <height>331</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit_E">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
    <addaction name="actionEdit"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuEdit_E"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As(S)</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit(Q)</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
